lib/plugins/aws/lib/naming.js

```diff
@@ -22,21 +21,17 @@ module.exports = {
     return /^ServiceEndpoint/;
   },
 
-  /*
-   * Stack Naming
-   */
+  // Stack
   getStackName() {
     return `${this.sdk.serverless.service.service}-${this.sdk.getStage()}`;
   },
 
-  getLogicalLogGroupName(functionName) {
-    return `${this.getNormalizedLambdaName(functionName)}LogGroup`;
+  getLogGroupLogicalId(functionName) {
+    return `${this.getNormalizedFunctionName(functionName)}LogGroup`;
```

Comment:
for me `lambda` means the actual lambda name that is deployed (service-stage-functionName) ... while function name is functionName only, because this name is extracted from `serverless.yml` which is provider agnostic